src/grass_gis_helpers/data_import.py

```diff
@@ -389,3 +390,61 @@ def import_local_xyz_files(
     if len(all_raster) > 0:
         imported_local_data = True
     return imported_local_data
+
+
+def import_local_vector_data(aoi_map, local_data_dir, rm_vectors, output):
+    """Imports vector data from local file path
+
+    Args:
+        aoi_map (str): name of vector map defining AOI
```

Comment:
Wollen wir uns hier auch angewöhnen nach dem `:` groß oder klein zu schreiben?

Reply:
Ja, habe ich nun bei beiden jeweils in groß geschrieben.